src/iot/explorer/iotNodes.ts

```diff
@@ -0,0 +1,52 @@
+/*!
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as vscode from 'vscode'
+import { localize } from '../../shared/utilities/vsCodeUtils'
+import { IotThing, IotClient } from '../../shared/clients/iotClient'
+import { AWSTreeNodeBase } from '../../shared/treeview/nodes/awsTreeNodeBase'
+import { ErrorNode } from '../../shared/treeview/nodes/errorNode'
+import { PlaceholderNode } from '../../shared/treeview/nodes/placeholderNode'
+import { makeChildrenNodes } from '../../shared/treeview/treeNodeUtilities'
+import { IotThingNode } from './iotThingNode'
+import { inspect } from 'util'
+import { IotThingFolderNode } from './iotThingFolderNode'
+import { IotCertsFolderNode } from './iotCertFolderNode'
+import { IotPolicyFolderNode } from './iotPolicyFolderNode'
+
+/**
+ * An AWS Explorer node representing S3.
+ *
+ * Contains buckets for a specific region as child nodes.
+ */
+export class IotNode extends AWSTreeNodeBase {
+    public readonly thingFolderNode: IotThingFolderNode
+    public readonly certFolderNode: IotCertsFolderNode
+    public readonly policyFolderNode: IotPolicyFolderNode
+
+    public constructor(private readonly iot: IotClient) {
+        super('IoT', vscode.TreeItemCollapsibleState.Collapsed)
```

Comment:
You will want to localize these label names as well.

Reply:
I didn't do it for this node since none of the other services (e.g. S3, Lambda, API Gateway) localized the name of their service either